step.yml

```diff
@@ -30,3 +30,12 @@ inputs:
       Newline (`\n`) separated list of go packages, to run the go test command against.
 
       Example: `github.com/my/step\ngithub.com/bitrise/step/tool`
+- code_coverage_path: $BITRISE_DEPLOY_DIR/coverage.txt
+  opts:
+    title: Destination path of the code coverage report file
+    summary: Destination path of the code coverage report file
+    description: |-
+      If this input is specified code coverage will be generated  
+      by running: go test -v -race -coverprofile=tmp/coverage/path -covermode=atomic against every package.  
+      Each package's coverage report will be appended to the file defined by this input.
+outputs:
```

Comment:
please remove if no output, however it would be more nice to have an output env, like $CODE_COVERAGE_REPORT_PATH rather than having an input to say `code_coverage_path: $BITRISE_DEPLOY_DIR/coverage.txt` It still can be the path of "$BITRISE_DEPLOY_DIR/coverage.txt" by default, but please have that path in an env as well. Reports are always good to have deployed, but with the exported env, you have the control for delete it from there - to not to deploy - or even to send it out and stuff. For example, if you need to set a path for the coverage report, then you need custom logic to parse that for example to send out in a slack step. So basically, I suggest you to have two output envs, a path(which points into the deplyo dir file), and the same but with the content. Both would be very handy, even for internal use.